config_panel.toml

```
@@ -10,8 +10,8 @@ services = ["__APP__-web", "__APP__-sidekiq", "__APP__-streaming"]
         [main.customization.max_toot_chars]
         ask = "Maximum allowed character count in a toot"
         type = "number"
+        example = "500"
         bind = ":/var/www/__APP__/live/live/.env.production"
```

```suggestion
        bind = ":/var/www/__APP__/live/.env.production"
```

(Randomly passing by, sounds like a typo)

---

Also note that it should be more semantic to use `__FINALPATH__` instead of `/var/www/__APP__`, but i'm not 100% sure this works